backend/app.py

```
@@ -68,7 +69,8 @@ def get_categories():
         conn.close()
         return jsonify({"categories": categories})
     except Exception as e:
-        return jsonify({"error": str(e)}), 500
+        app.logger.error(f"Error getting categories: {e}")
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)